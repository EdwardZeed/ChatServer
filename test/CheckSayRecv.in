REGISTER test5 ABC
LOGIN test5 ABC
CREATE channel4
JOIN channel4
SAY channel4 hi this is from test
q