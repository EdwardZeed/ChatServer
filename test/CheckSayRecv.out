RESULT REGISTER 1

RESULT LOGIN 1

RESULT CREATE channel4 1

RESULT JOIN channel4 1

RECV test5 channel4 hi this is from test

